Sources/PubNub/PubNubConfiguration.swift

```diff
@@ -96,8 +97,17 @@ public struct PubNubConfiguration: Hashable {
   /// [documentation](https://developer.apple.com/documentation/security/preventing_insecure_network_connections)
   /// for further details.
   public var useSecureConnections: Bool = true
-  /// Domain name used for requests ps.pndsn.com
-  public var origin: String = "ps.pndsn.com"
+  /// Full origin used for requests `subdomain`.`domain`
+  public var origin: String {
+    guard let sub = subdomain else {
+      return "ps.\(domain)"
+    }
+
+    return "\(sub).\(domain)"
+  }
+
+  public var subdomain: String?
+  public var domain: String = "pubnub.com"
```

Comment:
Why would this not be `"pndsn.com"` by default?

Reply:
There was a Slack thread about how we should change to use the `subscribe key` as the sub-domain, but `pndsn.com` doesn't currently have the wildcard to allow for this.  

I can revert the change until we get more clarification on what needs to be used.

Reply:
Yea it was a proposal for some improvement, need to qualify the details before we go changing.